client/src/shared/components/MessagePanelWrapper.tsx

```diff
@@ -5,9 +5,15 @@ import { GUN_POWDER, VANILLA_WHITE } from '@chattie/colors';
 const MessagePanelWrapper: FC<{
   borderLeft?: string;
   borderRight?: string;
+  className?: string;
   children: ReactNode;
-}> = ({ borderLeft = '', borderRight = '', children }) => (
-  <Stack direction="row" justifyContent="space-between">
+}> = ({ borderLeft = '', borderRight = '', className, children }) => (
+  <Stack
+    direction="row"
+    height="100%"
```

Comment:
I would recommend using 'vh' instead of '100%'.

Reply:
In this case I use `100%` instead of `100vh` or any other `viewport-percentage height` measure is to fill the component height that will be passed through `children`. Unfortunately I cannot use here `vh`, but overall good callout!